### PR TITLE
world's most insane build tool idea

### PR DESCRIPTION
**Pull request by hawkw**

### CODE REVIEW

**Comment by jamesmunns** [comment]
@hawkw this seems green now, should we merge it? if it works, hell yeah lets go

File: Cargo.toml [hunk 1]
```diff
@@ -45,7 +45,11 @@ default-members = [
     "source/*",
 
     # tools
+    # TODO(eliza): maybe we should skip manganese by default so we don't always build its bin deps?
     "tools/*",
+    # "tools/crowtty",
```
Comment (jamesmunns) [inline Cargo.toml:50]: Does this need to be cleaned up?

File: Cargo.toml [hunk 2]
```diff
@@ -161,3 +165,17 @@ rev = "101a4abaa19afdd131b334a16d92c9fb4909c064"
 [patch.crates-io.bbq10kbd]
 git = "https://github.com/hawkw/bbq10kbd"
 branch = "eliza/async"
+
+# necessary to avoid `esp32c3-hal` depending on `log` with version `=0.4.18`,
```
Comment (jamesmunns) [inline Cargo.toml:169]: oh no

File: oreboot-bt0.tar.gz
Comment (jamesmunns) [inline oreboot-bt0.tar.gz:1]: Did you mean to include these bin files?

**Review by jamesmunns — APPROVED** [review]
A few nits, but happy to ship and iterate!

File: oreboot-bt0.tar.gz
Comment (hawkw) [reply oreboot-bt0.tar.gz:1]: no i did not !

**Pull request merged**